taskt/Core/Documentation/DocumentationGeneration.cs

```diff
@@ -0,0 +1,238 @@
+                var classDescription = GetClassValue(commandClass, typeof(Description));
+                var usesDescription = GetClassValue(commandClass, typeof(UsesDescription));
+                var commandName = instantiatedCommand.SelectionName;
+
+                stringBuilder = new StringBuilder();
+
+                //create string builder to build markdown document and append data
+                stringBuilder.AppendLine("<!--TITLE: " + commandName + " Command -->");
+                stringBuilder.AppendLine("<!-- SUBTITLE: a command in the " + groupName + " group. -->");
+
+                stringBuilder.AppendLine("[Go To Automation Commands Overview](/automation-commands)");
+
+                stringBuilder.AppendLine(Environment.NewLine);
+                stringBuilder.AppendLine("# " + commandName + " Command");
+                stringBuilder.AppendLine(Environment.NewLine);
+
+                //append more
+                stringBuilder.AppendLine("## What does this command do?");
+                stringBuilder.AppendLine(classDescription);
+                stringBuilder.AppendLine(Environment.NewLine);
+
+                //more
+                stringBuilder.AppendLine("## When would I want to use this command?");
+                stringBuilder.AppendLine(usesDescription);
+                stringBuilder.AppendLine(Environment.NewLine);
+
+                //build parameter table based on required user inputs
+                stringBuilder.AppendLine("## Command Parameters");
+                stringBuilder.AppendLine("| Parameter Question   	| What to input  	|  Sample Data 	| Remarks  	|");
+                stringBuilder.AppendLine("| ---                    | ---               | ---           | ---       |");
+
+                //loop each property
+                foreach (var prop in commandClass.GetProperties().Where(f => f.Name.StartsWith("v_")).ToList())
+                {
+                    //pull attributes from property
+                    var commandLabel = GetPropertyValue(prop, typeof(PropertyDescription));
+                    var helpfulExplanation = GetPropertyValue(prop, typeof(InputSpecification));
+                    var sampleUsage = GetPropertyValue(prop, typeof(SampleUsage));
+                    var remarks = GetPropertyValue(prop, typeof(Remarks));
+
+                    //append to parameter table
+                    stringBuilder.AppendLine("|" + commandLabel + "|" + helpfulExplanation + "|" + sampleUsage + "|" + remarks + "|");
+                }
+
+                stringBuilder.AppendLine(Environment.NewLine);
+
+                stringBuilder.AppendLine("## Developer/Additional Reference");
+                stringBuilder.AppendLine("Automation Class Name: " + commandClass.Name);
+                stringBuilder.AppendLine("Parent Namespace: " + commandClass.Namespace);
+                stringBuilder.AppendLine("This page was generated on " + DateTime.Now.ToString("MM/dd/yy hh:mm tt"));
+
```

Comment:
Minor change, please remove some of the whitespace between the stringbuilder.AppendLine()s. Both here and further down.

Reply:
done